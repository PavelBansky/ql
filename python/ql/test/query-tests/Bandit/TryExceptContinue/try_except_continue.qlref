Bandit/try_except_continue.ql