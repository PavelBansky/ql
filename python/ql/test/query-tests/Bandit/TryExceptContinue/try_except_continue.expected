| Test.py:6:9:6:16 | Continue | Try, Except, Continue detected. |
| Test.py:14:9:14:16 | Continue | Try, Except, Continue detected. |
| Test.py:22:9:22:16 | Continue | Try, Except, Continue detected. |
